lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -137,8 +137,8 @@ sub cache_assets {
         if ($asset_request->enqueue) {
             log_debug("Downloading " . $asset_uri . " - request sent to Cache Service.", channels => 'autoinst');
             update_setup_status and sleep 5 until $asset_request->processed;
-            log_debug("Download of " . $asset_uri . " processed", channels => 'autoinst');
-            log_debug($asset_request->output,                     channels => 'autoinst');
+            my $msg = "Download of $asset_uri processed";
+            if (my $output = $assert_request->output) { $msg .= ": $output" };
```

Comment:
Just the `log_debug` call missing now. 😉

Reply:
:D Yeah if a certain tools team would not have decided that it would be an awesome idea to make testing harder by removing sqlite support I would have actually tested that! ;) Working on tests now

Reply:
I wouldn't know, that was before i joined the team. But i completely agree with the decision, all you get by using SQLite for tests and PostgreSQL in production is an unrealistic test environment. Not worth the trouble.

Reply:
updated PR. Added the `log_debug` call and a trivial test to just cover some code lines and merely check for compile errors. I consider this mergeable as is. I still have the plan to work on more proper tests eventually but with lower priority.